contracts/src/StakeTableV2.sol

```diff
@@ -84,6 +89,12 @@ contract StakeTableV2 is StakeTable, PausableUpgradeable, AccessControlUpgradeab
     /// @notice Maximum commission increase allowed per increase (in basis points)
     uint16 public maxCommissionIncrease;
 
+    /// @notice Total stake in active (not marked for exit) validators in the contract
+    uint256 public totalValidatorStake;
```

Comment:
I'm not sure about the naming because I couldn't tell just from the names what the variables track. 

I feel like it would be best to have pendingWithdrawals, pendingExits, activeStake and totalTracked but this would be 4 storage edits for each delegation which is quite expensive.

Reply:
that's fair at first delegation, in this case, i used the average fees to compare. 
If we only store totalValidatorStake which i'm thinking to rename to `totalEffectiveStake` or `totalConsensusStake`, the (median) gas increase for a delegation is 25%.

I've made the change to only track `totalValidatorStake` here, if you agree, i'll merge it into this pr